front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1,3688 +1,121 @@
+        // so we need to do it manually with this function and enhanced endpoint
+        url: `/v2/train_schedule/simulation_summary?infra=${queryArg.infra}&${formatURLWithIds(queryArg.ids)}`,
+        method: 'GET',
+      }),
+    },
+    getV2TrainSchedule: {
+      query: (queryArg) => ({
+        // We currently can't build the url path the way we want with rtk query with the regular endpoint
+        // so we need to do it manually with this function and enhanced endpoint
+        url: `/v2/train_schedule?${formatURLWithIds(queryArg.ids)}`,
+        method: 'GET',
+      }),
+    },
+    postV2TrainScheduleProjectPath: {
+      query: (queryArg) => ({
+        // We currently can't build the url path the way we want with rtk query with the regular endpoint
+        // so we need to do it manually with this function and enhanced endpoint
+        url: `/v2/train_schedule/project_path?infra=${queryArg.infra}&${formatURLWithIds(queryArg.ids)}`,
+        method: 'POST',
+        body: queryArg.projectPathInput,
+      }),
+      // As we always use all get trainschedule v2 endpoints after updating the timetable,
+      // we don't want to invalidate the trainschedulev2 tag here to preven multiple calls
+      invalidatesTags: [],
+    },
+    deleteV2TrainSchedule: {
+      query: (queryArg) => ({
+        url: `/v2/train_schedule/`,
+        method: 'DELETE',
+        body: queryArg.body,
+      }),
+      // As we always use all get trainschedule v2 endpoints after updating the timetable,
+      // we don't want to invalidate the trainschedulev2 tag here to preven multiple calls
+      invalidatesTags: ['timetablev2', 'scenariosv2'],
+    },
+    postV2TimetableByIdTrainSchedule: {
+      query: (queryArg) => ({
+        url: `/v2/timetable/${queryArg.id}/train_schedule/`,
+        method: 'POST',
+        body: queryArg.body,
+      }),
+      // As we always use all get trainschedule v2 endpoints after updating the timetable,
+      // we don't want to invalidate the trainschedulev2 tag here to preven multiple calls
+      invalidatesTags: ['timetablev2', 'scenariosv2'],
+    },
+    // Invalidate the children count and last update timestamp
+    postProjectsByProjectIdStudies: {
+      invalidatesTags: ['studies', 'projects'],
+    },
+    patchProjectsByProjectIdStudiesAndStudyId: {
+      invalidatesTags: ['studies', 'projects'],
+    },
+    deleteProjectsByProjectIdStudiesAndStudyId: {
+      invalidatesTags: ['studies', 'projects'],
+    },
+    postProjectsByProjectIdStudiesAndStudyIdScenarios: {
+      invalidatesTags: ['scenariosv2', 'studies', 'projects'],
+    },
+    patchProjectsByProjectIdStudiesAndStudyIdScenariosScenarioId: {
+      invalidatesTags: ['scenariosv2', 'studies', 'projects'],
+    },
+    deleteProjectsByProjectIdStudiesAndStudyIdScenariosScenarioId: {
+      invalidatesTags: ['scenariosv2', 'studies', 'projects'],
```

Comment:
For all : I don't think you need to invalidate `projects` for these operations as project cards list only the studies count (not the scenarios). And as they are v1 endpoints, they should invalidate `scenarios` rathan than `scenariosv2`.
For put : you don't need the `studies` tag as the study card needs only the scenario count info

```suggestion
      invalidatesTags: ['scenarios', 'studies'],
    },
    deleteProjectsByProjectIdStudiesAndStudyIdScenariosScenarioId: {
      invalidatesTags: ['scenarios', 'studies'],
```

Reply:
> For all : I don't think you need to invalidate projects for these operations as project cards list only the studies count (not the scenarios).

The "last updated at" time still needs to be bumped though.

> And as they are v1 endpoints, they should invalidate scenarios rathan than scenariosv2.

Good point!

> For put : you don't need the studies tag as the study card needs only the scenario count info

I think it's the same case as above for the "last updated at" thing?

Reply:
Oh yes, you're right. I thought updating a scenario wasn't supposed to update the "last updated" for projects. If you add a train in a scenario, it still doesn't update this property, even with your modifications, maybe it's not supposed to. But updating the scenario's description does properly update the last_updated_at from projects so we should leave it as you said.

Reply:
I've updated the patch to invalidate scenarios rather than scenariosv2 when using the v1 endpoints.

>  If you add a train in a scenario, it still doesn't update this property, even with your modifications, maybe it's not supposed to.

Right, adding a train in a scenario doesn't bump the "last updated at" timestamp. However it does bump the number of trains count in the scenario, so that's why I invalidate scenarii there.